config/routes.rb

```diff
@@ -9,6 +9,13 @@
   #   e.g. https://scholarsphere.psu.edu/files/gm80hv36p
   get '/files/:id', to: redirect('/concern/generic_works/%{id}')
 
+  post '/uploads', to: 'sufia/uploads#create'
+  # This is a hack that is required because the rails form the uploader is on
```

Comment:
You're either missing a word or have an extra one here: `because the rails form the uploader`